src/Converter/JsonConverter.php

```diff
@@ -833,7 +833,7 @@ public function convertObject(array $data = null)
         } else {
             $properties = $this->convertProperties(
                 $data[JSONConstants::JSON_OBJECT_PROPERTIES] ?? [],
-                (array) $data[JSONConstants::JSON_OBJECT_PROPERTIES_EXTENSION] ?? []
+                $data[JSONConstants::JSON_OBJECT_PROPERTIES_EXTENSION] ?? []
```

Comment:
It would probably be better to avoid potential null by using parenthesis: `(array)
 ($data[JSONConstants::JSON_OBJECT_PROPERTIES_EXTENSION] ?? [])` would cast correctly, ensuring an array is passed as second argument to convertProperties().

Reply:
I agree